packages/react/src/themes/teams/components/Menu/menuStyles.ts

```diff
@@ -11,19 +11,22 @@ import {
 type MenuPropsAndState = MenuProps & MenuState
 
 export default {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => {
-    const { iconOnly, fluid, pointing, pills, primary, underlined, vertical, submenu } = p
+  root: ({ props: p, variables: v, theme }): ICSSInJSStyle => {
+    const { iconOnly, fluid, pointing, pills, primary, color, underlined, vertical, submenu } = p
+    const colorScheme = v.colorScheme[color ? color : primary ? 'primary' : 'default']
 
     return {
       display: 'flex',
       minHeight: pxToRem(24),
       margin: 0,
       padding: 0,
+      color: v.color || colorScheme.backgroundDefault,
+      backgroundColor: v.backgroundColor || 'inherit',
       listStyleType: 'none',
       ...(iconOnly && { alignItems: 'center' }),
       ...(vertical && {
         flexDirection: 'column',
-        backgroundColor: v.verticalBackgroundColor,
+        backgroundColor: v.verticalBackgroundColor || theme.siteVariables.colors.white,
```

Comment:
why don't we take this color from `colorScheme`?

Reply:
Actually we should take this color from the variable (my bad for introducing color directly in the styles :\) The vertical menu's should always have solid background, provided by the theme. The color prop is changing the border's, the background of the active menu items etc. Good catch though..